### PR TITLE
Small refactor

### PR DESCRIPTION
**Pull request by cabello**
I did a series of small changes, please see if you like it and it makes sense.

### CODE REVIEW

**Comment by chesterbr** [comment]
I did, but I only saw it now :-(

**Comment by chesterbr** [comment]
(forgot I disabled auto-follow projects)

File: app/src/js/app.js [hunk 1]
```diff
@@ -2,16 +2,20 @@ internet = require('internet');
...
-    internet.getRoutes(loc.coords.latitude, loc.coords.longitude, function(routes) {
-      bluetooth.sendRoutes(routes);
-    })
-  });
-}
-
-Pebble.addEventListener('ready', openRoutesList);
+function onPebbleReady(e) {
+  openRoutesList();
+};
```
Comment (chesterbr) [inline app/src/js/app.js]: Is the idea behind this split to enable the reader to get the events in order? That is, "when the Pebble is ready, open the routes list"?

**Comment by chesterbr** [comment]
Changes make total sense (even more now that we have error flows). I've conciliated them with the current code in #2, so I'll close this one. Thank you and my apologies for the oversight!

**Comment by cabello** [comment]
Awesome, glad you liked my small collaboration.

**Pull request closed without merging**